resources/puppetlabs/lein-ezbake/staging-templates/project_data.yaml.mustache

```diff
@@ -36,12 +36,16 @@ templates:
     target: ext/{{{project}}}.logrotate-legacy.conf
   - source: ext/ezbake.tmpfiles.conf.erb
     target: ext/{{{project}}}.tmpfiles.conf
+  - ext/fpm.rb.erb
+  - ext/redhat/preinst.erb
+  - ext/redhat/postinst.erb
+  - ext/redhat/prerm.erb
+  - ext/redhat/postrm.erb
```

Comment:
It generates an entirely new one with these files and all the parameters that are passed to the fpm command. I'll grab a spec file and what the fpm command ends up looking like for deb/rpm to share here.